src/backend/backend/api/routes/agents.py

```diff
@@ -0,0 +1,48 @@
+from uuid import UUID
+
+from fastapi import APIRouter, status
+from fastapi.responses import FileResponse
+
+from backend.api.deps import AgentServiceDep
+from backend.schemas import AgentConfig, AgentCreateRequest, AgentSummary
+from backend.services.exceptions import AgentNotFoundError
+
+router = APIRouter()
+
+
+def agent_exception_mappings() -> dict[type[AgentNotFoundError], int]:
+    return {
+        AgentNotFoundError: status.HTTP_404_NOT_FOUND,
+    }
+
+
+@router.post("/", status_code=status.HTTP_201_CREATED)
+async def create_agent(
+    service: AgentServiceDep,
+    agent_create_request: AgentCreateRequest,
+) -> AgentSummary:
+    return await service.create_agent(agent_create_request)
```

Comment:
Since we're not returning the full object, we can add the `Location` header to point to where it can be found.

```suggestion
    agent_create_request: AgentCreateRequest,
    request: Request,
    response: Response,
) -> AgentSummary:
    agent_summary = await service.create_agent(agent_create_request)
    url = request.url_for(get_agent.__name__, agent_id=agent_summary.id)
    response.headers["Location"] = f"{url}"
    return agent_summary
```